profiler/src/main/java/com/splunk/opentelemetry/profiler/context/SpanContextualizer.java

```diff
@@ -101,6 +101,10 @@ StackToSpanLinkage link(Instant time, String rawStack, long threadId, RecordedEv
     return new StackToSpanLinkage(time, rawStack, eventName, spanLinkage);
   }
 
+  public SpanLinkage link(long threadId) {
```

Comment:
Bummer now that one of the `link()` methods can return null and the other doesn't. I think it would be nice if they were consistent. ..but I understand that this is probably slightly more efficient (and saves an allocation in the hot path).

Reply:
Oh, I can return `SpanLinkage.NONE` from here -- I just noticed that.